robot-custom-integration/sl_python_robot/SLListener.py

```diff
@@ -64,16 +70,16 @@ def create_test_session(self):
 
         if not response.ok:
             print(
-                f'>>>> {SEALIGHTS_LOG_TAG} Failed to open Test Session (Error {response.status_code}),'
+                f'{SEALIGHTS_LOG_TAG} Failed to open Test Session (Error {response.status_code}),'
                 f' disabling Sealights Listener')
         else:
             res = response.json()
             self.test_session_id = res["data"]["testSessionId"]
-            print(f'>>>> {SEALIGHTS_LOG_TAG} Test session opened, testSessionId: {self.test_session_id}')
+            print(f'{SEALIGHTS_LOG_TAG} Test session opened, testSessionId: {self.test_session_id}')
 
     def get_excluded_tests(self):
         recommendations = requests.get(f'{self.get_session_url()}/exclude-tests', headers=self.get_header())
-        print(f'Recommendations: {recommendations.status_code}')
+        print(f'{SEALIGHTS_LOG_TAG} Retrieving Recommendations: {"OK" if recommendations.ok else "Error {recommendations.status_code}"}')
```

Comment:
word "Recommendations" should not be capitalized as it's not the first one in the sentence anymore.